packages/geo-diary/src/core/__tests__/app.tsx

```diff
@@ -6,6 +6,23 @@ import { render, unmountComponentAtNode } from 'react-dom'
 jest.mock('../router')
 
 describe('App', () => {
+  beforeAll(() => {
+    // @ts-ignore
+    delete global.window.location
+    // @ts-ignore
+    global.window = Object.create(window)
+    // @ts-ignore
+    global.window.location = {
```

Comment:
Can you do `jest.spyOn(global, 'window', () => ({ //mock here })`

Reply:
The `window.location` in the test env don't have reload function. Seems like we can't mock not exist function using jest